### DISCUSSION TITLE
Snap&Run for free?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1087] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My free trial expired, now I can't use Snap&Read or Snap&Run.

**Comment by S-PScripts** [comment]
cant you get another free trial

**Reply by AshtonDavies** [reply, marked as answer]
Or just use the DJ account provided by your school if they happen to be allowing the extension in the first place.

**Comment by JustTrafik** [comment]
are you fucking retarded
